doc/nrf/ug_thread_tools.rst

```diff
@@ -274,16 +274,17 @@ The interaction is possible using commands proper to wpanctl, a module installed
 Installing wpantund
 ===================
 
-To ensure that the interaction with the samples works as expected, install the version of wpantund that has been used for testing the |NCS|.
```

Comment:
Why are we getting rid of the general installation and version instructions? They were requested not long ago.
We can still have a note stating that you need a different version for USB.

Reply:
Update the wpantund commit to: bf45115f41ba2b8029eda174be2b93dea73b9261 please